pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -7473,6 +7808,8 @@ def scale_model(model, scaling_factor=1000000):
     model.scaling_factor[model.TotalTruckingCost] = 1 / scaling_factor
     model.scaling_factor[model.TotalTruckingVolume] = 1 / scaling_factor
     model.scaling_factor[model.TreatmentFeedBalance] = 1 / scaling_factor
+    model.scaling_factor[model.TreatmentFeedTechLHS] = 1 / scaling_factor
+    model.scaling_factor[model.TreatmentFeedTechRHS] = 1 / scaling_factor
     model.scaling_factor[model.TreatmentBalance] = 1 / scaling_factor
     model.scaling_factor[model.TreatedWaterBalance] = 1 / scaling_factor
     model.scaling_factor[model.ResidualWaterBalance] = 1 / scaling_factor
```

Comment:
We should also scale the objective for the emissions right?

Reply:
@JavalVyas2000 the emissions objective is implemented different than the other objective functions in that it uses a Pyomo expression instead of a dedicated variable. I think you can only scale Params and Vars in Pyomo - I'm not sure if there's a good way to scale an Expression.

Reply:
I believe we can scale the expressions as well. Not sure we need to look into it, but we can for sure scale constraints so I dont think why we could not scale the expression